src/plugins/plugin.ts

```diff
@@ -23,10 +25,12 @@ import { RedPacketPluginDefine } from './Wallet/define'
 import type { PostInfo } from '../social-network/PostInfo'
 import { StorybookPluginDefine } from './Storybook/define'
 import { FileServicePluginDefine } from './FileService/define'
+import { TraderPluginDefine } from './Trader/define'
 import { Flags } from '../utils/flags'
 plugins.add(GitcoinPluginDefine)
 plugins.add(RedPacketPluginDefine)
 if (Flags.file_service_enabled) plugins.add(FileServicePluginDefine)
+plugins.add(TraderPluginDefine)
```

Comment:
Maybe we also need a flag for it. Does it ready for mobile? @neruthes

Reply:
Not for mobile right now. Focus on desktop at this moment.